test/invoice_repository_test.rb

```diff
@@ -5,7 +5,7 @@
 class InvoiceRepositoryTest < MiniTest::Test
   attr_reader :invoice_repo
   def setup
-    @invoice_repo = InvoiceRepository.new('test/support/sample_invoices.csv')
+    @invoice_repo = InvoiceRepository.new('test/support/invoices.csv')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.